### ISSUE TITLE
Playlist subsets removed?

### ISSUE THREAD

**Issue by lewkaj** [shaked6540/YoutubePlaylistDownloader#20]
Was that feature removed or moved somewhere else after settings rework? I can't find it anymore.

**Comment by shaked6540** [comment]
yes, it was removed because I planned to add a manual selection of videos you want\dont want but I didn't have time to implement that. Thinking back, it was a bad idea to remove it before I added the manual selection
